src/components/EditImageModal/index.jsx

```diff
@@ -1,44 +1,62 @@
 import React from 'react';
-import { Button, CheckBox, Icon, InputText, Modal } from '@edx/paragon';
+import { Button, CheckBox, Icon, InputText, Modal, StatusAlert } from '@edx/paragon';
+import classNames from 'classnames';
 import FontAwesomeStyles from 'font-awesome/css/font-awesome.min.css';
 
 import messages from './displayMessages';
-import './EditImageModal.scss';
+import styles from './EditImageModal.scss';
 import WrappedMessage from '../../utils/i18n/formattedMessageWrapper';
 import rewriteStaticLinks from '../../utils/rewriteStaticLinks';
 
 const LOADING_SPINNER_DELAY = 1000; // in milliseconds
 
+const imageDescriptionHelpLink = 'http://edx.readthedocs.io/projects/edx-partner-course-staff/en/latest/accessibility/best_practices_course_content_dev.html#use-best-practices-for-describing-images';
+const imageDescriptionID = 'imageDescription';
+const imageSourceID = 'imageSource';
+
+const initialEditImageModalState = {
```

Comment:
This allows an easier reset of state when modal is opened via the "handleModalOpen" function.